cypress/support/utils.ts

```diff
@@ -50,22 +50,26 @@ export async function runTest(test: string) {
         console.log("Running with local db");
       }
 
-      return new Promise(async (resolve, reject) => {
+      const promise = new Promise(async (resolve, reject) => {
```

Comment:
We could go a little bit simpler

```suggestion
      await new Promise(async (resolve, reject) => {
```